packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -4737,15 +4739,18 @@ export class WalletService implements IWalletService {
   txConfirmationSubscribe(opts, cb) {
     if (!checkRequired(opts, ['txid'], cb)) return;
 
-    const sub = TxConfirmationSub.create({
-      copayerId: this.copayerId,
-      walletId: this.walletId,
-      txid: opts.txid,
-      amount: opts.amount,
-      isCreator: true
-    });
+    const txids = _.isArray(opts.txid) ? opts.txid : [opts.txid];
+    for (let i = 0; i < txids.length; i++) {
```

Comment:
Do we need the (i)ndex below or could this be simplified to `for (const txid of txids)`? - total nitpick